pkgs/tools/admin/awscli2/default.nix

```diff
@@ -1,22 +1,14 @@
-{ lib, python3, groff, less, fetchFromGitHub, fetchpatch }:
+{ lib, python3, groff, less, fetchFromGitHub }:
 let
   py = python3.override {
     packageOverrides = self: super: {
-      awscrt = super.awscrt.overridePythonAttrs (oldAttrs: rec {
```

Comment:
`awscrt` is still required and pinned to an older version than is in `nixpkgs`:
https://github.com/aws/aws-cli/blob/29ee73bc7bb1b0538cb315313cf149b5215ee2dc/setup.cfg#L38
https://github.com/NixOS/nixpkgs/blob/3adcef25692be8ed9575d01766c3ab7b1a26f2cb/pkgs/development/python-modules/awscrt/default.nix#L5

So we either need to leave this override in place, patch `awscli2` to be more relaxed about the version of `awscrt`, or ensure updates to both `awscrt` and `awscli2` are in sync (which currently would mean downgrading the package on master).

Reply:
With this it builds for me:

```nix
      awscrt = super.awscrt.overridePythonAttrs (oldAttrs: rec {
        version = "0.12.4";
        src = self.fetchPypi {
          inherit (oldAttrs) pname;
          inherit version;
          sha256 = "sha256:1cmfkcv2zzirxsb989vx1hvna9nv24pghcvypl0zaxsjphv97mka";
        };
      });
```

Reply:
Might make sense to keep it pinned. Looks like [botocore also requires awscrt==0.12.4](https://github.com/boto/botocore/blob/3ff30ef929da20bc83a1f21d1bd3ab6e408648fe/setup.cfg#L9). 

It's 0.12.5 on the [the development branch of botcore](https://github.com/boto/botocore/blob/f272c5b277bc049b221bca40d10ca967d7161dd8/setup.cfg#L11).

Reply:
Thx for the input :+1: